DatabaseTemplate/Modules/create-database-template.psm1

```diff
@@ -477,7 +477,7 @@ function Backup-DatabaseTemplate {
 
     if ($config.engine -eq 'SQLServer') {
         $params = @{
-            csb               = Convert-CommonDbCSBtoSqlCSB $csb
+            csb               = $csb
```

Comment:
![image](https://user-images.githubusercontent.com/34613894/236507507-a1545aee-11fa-45ab-a804-1fb1b91cb84b.png)
This file is used various database template packages ,so you need to run the database template workflow to update the package version for all database template packages